### ISSUE TITLE
Use Drivah

### ISSUE THREAD

**Issue by tarekziade** [elastic/connectors#1023]
Steal what @seanstory  is doing in https://github.com/elastic/data-extraction-service/pull/7  :) 

cc @markjhoy

**Comment by danajuratoni** [comment]
@DianaJourdan @seanstory is this ticket still needed?

**Comment by seanstory** [comment]
It's not "needed" but it would help us clean up some of the bespoke build stuff that we're doing. Maybe a task we can hand off to the Developer Productivity team, to utilize Buidah and Drivah? CC @markjhoy

**Comment by markjhoy** [comment]
Yep, @seanstory - we have an item for "Buildkite Optimizations" on [our planning roadmap](https://docs.google.com/spreadsheets/d/1CSq0bF2Zropqp9VqJvI1tVZ4KmVaOU9wsL7QEpGjdmY/edit#gid=0) that I think this could fit in nicely. No concrete thoughts planned out yet, but this makes sense as part of that.

**Comment by seanstory** [comment]
@markjhoy do we still need this, or is it redundant to other tickets on other backlogs?

**Comment by markjhoy** [comment]
It _might_ be redundant? I'll have to look closer, and it may be the case that using Drivah may work best.

**Comment by seanstory** [comment]
Keep me posted. Or just close this issue if you determine that we've got this work covered elsewhere.
